### PR TITLE
Add script for demoing how to measure your build

### PR DESCRIPTION
**Pull request by mthmulders**

### CODE REVIEW

File: demo-1-measure.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  jaegertracing/all-in-one:1.54
+
+echo Navigate to http://localhost:16686/
+read
+
+pushd shiro
+
+cp ../extensions.xml ./.mvn/
+
+mvn3 --file pom.xml -Dotel.traces.exporter=otlp verify --projects '!org.apache.shiro.integrationtests:shiro-its-jakarta-ee'
```
[the remark this answers is not in the document]
Comment (mthmulders) [reply demo-1-measure.sh:26]: if `mvn3` is defined, we're good; if it's not defined, define it as an alias to `mvn`.

**Comment by bmarwell** [comment]
Hmm. Do we need to clean up between the builds?

**Comment by mthmulders** [comment]
> Hmm. Do we need to clean up between the builds?

I don't think we should. Download time can be a huge influence on build speed, which is why on Azure DevOps and GitHub Actions and the like people often use caching. Also, conference WiFi is not always very reliable, and we don't want that to break our demos. So I would argue to keep downloads out of the equation as much as we can.

**Comment by bmarwell** [comment]
No, I mean reset the shiro checkout, not clean the .m2/repository.

**Comment by mthmulders** [comment]
Need to rebase this on #8 once that's merged.

**Pull request merged**